docs-ref-conceptual/manage-cloud-sets-azure-cli.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Managing Cloud Sets with Azure CLI 2.0
+description: Create, log in, and manage different cloud sets with the Azure CLI 2.0.
+keywords: Azure CLI 2.0, Azure, cloud sets, datacenters
+cloud: ???

```

Comment:
@rloutlaw - Not sure what this tag value should be. Do we have a list of them available?